README.md

```diff
@@ -1,15 +1,17 @@
 # GothicLLamasBE
-Team GothicLlama back end repo.
+Team GothicLlamas' back end Java API repo.
 
 How to start the application
 ---
 
-1. Run `mvn clean install` to build your application
-2. Navigate to GothicLlamasAPI folder.
+1. Navigate to GothicLlamasAPI folder
+2. Run `mvn clean install` to test and build your application
 3. Start application with `java -jar target/GothicLlamas-1.0-SNAPSHOT.jar server config.yml`
```

Comment:
`config.yml` isn't in the codebase and there are no instructions here on what should be in it. Doesn't this mean that following these instructions *will not work*?

Reply:
Ah good point, I'll add a few lines in here detailing the contents of the file

Reply:
I was wondering this before and complete forgot until this comment, it will likely be the same with the db properties file, is there any safe way to say how these files should exist without giving away the information?